bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
@omkreddy 
I agree the lack of consistency is a bit suspect, but I ran this change by a few committers, and no one seemed to mind. I also didn't see any obvious drawbacks given that it's in its own jar.

As for why I used argparse4j, I think it was roughly:
- partly ease of use and cleaner api, particularly when using in java
- partly curiosity
- partly when writing the java tool, I was unable to get joptsimple to parse -1 properly as an integer as opposed to a flag. This was actually pretty surprising, so maybe I missed something obvious